### PR TITLE
Creating a example of storage reuse with multiple datasets

### PR DESCRIPTION
**Pull request by andersonpablosilva**
Creating a new notebook demonstrating storage reuse in a scenario with multiple datasets and common files between them.

### CODE REVIEW

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.4"
+  },
+  "pycharm": {
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb:329]: We don't need this pycharm related configuration here.

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### To start using the ml-git api we need to import it into our script"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb:30]: We need to clean all execution_count:
```
"execution_count": null,
```

Comment (Mun1z) [reply docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb]: Done.

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb (hunk 1 above)
Comment (Mun1z) [reply docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb]: Done.

**Review by Mun1z — APPROVED** [review]

**Review by gilMars — APPROVED** [review]

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+    "## Storage reuse with multiple datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### This notebook describes a storage reuse scenario with multiple datasets containing common datapoints.\n",
+    "\n",
+    "To reuse space in the bucket, when the same file is used in multiple datasets, this file will only be added to the bucket only once, to exemplify the use two entities will be created, the people entity contains 10 images with faces of people, while famous entity contains 7 images with faces of famous people, of these 7 images, 5 of them are contained in the entity people.\n",
```
Comment (diego-pedroso) [inline docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb:16]:
```suggestion
    "When the same file is used in multiple datasets, that file will be added to the bucket only once, in order to optimize the space usage in the bucket. To exemplify this use case, two entities will be created: the people entity contains 10 images with faces of people, while famous entity contains 7 images with faces of famous people, being 5 of them also contained in the people entity.\n",
```

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+    "\n",
+    "This way, when sending the files to the repository, the 5 images that are being used in the two entities will not be duplicated in the bucket, the two entities will refer to the same image stored in the bucket."
```
Comment (diego-pedroso) [inline docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb:18]:
```suggestion
    "This way, when sending the files to the repository, the 5 images that are being used in the two entities will not be duplicated in the bucket. The two entities will refer to the same image stored in the bucket."
```

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb (hunk 3 above)
Comment (Mun1z) [reply docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb]: Suggestion accepted.

File: docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb (hunk 4 above)
Comment (Mun1z) [reply docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb]: Suggestion accepted.

**Review by diego-pedroso — APPROVED** [review]

**Pull request merged**